notebooks/nightly_report/nightly_report_convergence.ipynb

```diff
@@ -1,4 +1,4 @@
~{
```

Comment:
<p>It would be great to use a different color palette for first panel (plotting different elevations) and subsequent plots (where we plot AOS FWHM, donut blur, etc) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>